src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/DWMColorization.cs

```diff
@@ -0,0 +1,682 @@
+                blue = (int)(p * 255.0f + 0.5f);
+
+                break;
+            case 2:
+                red = (int)(p * 255.0f + 0.5f);
+                green = (int)(brightness * 255.0f + 0.5f);
+                blue = (int)(t * 255.0f + 0.5f);
+
+                break;
+            case 3:
+                red = (int)(p * 255.0f + 0.5f);
+                green = (int)(q * 255.0f + 0.5f);
+                blue = (int)(brightness * 255.0f + 0.5f);
+
+                break;
+            case 4:
+                red = (int)(t * 255.0f + 0.5f);
+                green = (int)(p * 255.0f + 0.5f);
+                blue = (int)(brightness * 255.0f + 0.5f);
+
+                break;
+            case 5:
+                red = (int)(brightness * 255.0f + 0.5f);
+                green = (int)(p * 255.0f + 0.5f);
+                blue = (int)(q * 255.0f + 0.5f);
+
+                break;
+        }
+
+        return (red, green, blue);
+    }
+
+    /// <summary>
+    /// Converts the color values stored as HSL to RGB.
+    /// </summary>
+    private static (int R, int G, int B) FromHslToRgb(float hue, float saturation, float lightness)
+    {
+        if (AlmostEquals(saturation, 0, 0.01f))
+        {
+            var color = (int)(lightness * _byteMax);
+
+            return (color, color, color);
+        }
+
+        lightness /= 100f;
+        saturation /= 100f;
+
+        var hueAngle = hue / 360f;
+
+        return (
+            CalcHslChannel(hueAngle + 0.333333333f, saturation, lightness),
+            CalcHslChannel(hueAngle, saturation, lightness),
+            CalcHslChannel(hueAngle - 0.333333333f, saturation, lightness)
+        );
+    }
+
+    /// <summary>
+    /// HSV representation models how colors appear under light.
+    /// </summary>
+    /// <returns><see langword="float"/> hue, <see langword="float"/> saturation, <see langword="float"/> brightness</returns>
+    private static (float Hue, float Saturation, float Value) ToHsv(Color color)
+    {
+        int red = color.R;
+        int green = color.G;
+        int blue = color.B;
+
+        var max = Math.Max(red, Math.Max(green, blue));
+        var min = Math.Min(red, Math.Min(green, blue));
+
+        var fDelta = (max - min) / _byteMax;
+
+        float hue;
+        float saturation;
+        float value;
+
+        if (max <= 0)
+        {
+            return (0f, 0f, 0f);
+        }
+
+        saturation = fDelta / (max / _byteMax);
+        value = max / _byteMax;
+
+        if (fDelta <= 0.0)
+        {
+            return (0f, saturation * 100f, value * 100f);
+        }
+
+        if (max == red)
+        {
+            hue = ((green - blue) / _byteMax) / fDelta;
+        }
+        else if (max == green)
+        {
+            hue = 2f + (((blue - red) / _byteMax) / fDelta);
+        }
+        else
+        {
+            hue = 4f + (((red - green) / _byteMax) / fDelta);
+        }
+
+        if (hue < 0)
+        {
+            hue += 360;
```

Comment:
I do not think `hue` should add 360, and I think `hue` should add `6` here. Because the Line 495 will return `hue * 60f``.

Reply:
We have removed the calculation of accent color variations and hence this is no longer in use.